src/rules/no-export-load-in-svelte-module-in-kit-pages.ts

```diff
@@ -0,0 +1,52 @@
+import type * as ESTree from "estree"
+import { createRule } from "../utils"
+import fs from "fs"
+
+const hasSvelteKit = (() => {
+  try {
+    const packageJson = JSON.parse(fs.readFileSync("./package.json", "utf8"))
+    return Boolean(packageJson.dependencies["@sveltejs/kit"])
+  } catch (_e) {
+    return false
+  }
+})()
```

Comment:
If a user doesn't use SvelteKit, I don't want to show an error, so I added to check `package.json`.
Is it the correct way?